dist/routable-component.js

```diff
@@ -455,17 +455,17 @@ var routableComponent =
 	  }, {
 	    key: 'event',
 	    value: function event() {
-	      if (this._current_from && this._current_controller) {
+	      if (this._currentFrom && this._currentController) {
 	        if (arguments.length > 2) throw new Error('arguments too long');
-	        this.event_on_from_controller.apply(this, arguments);
-	      } else if (this._current_from) {
+	        this.eventOnFromController.apply(this, arguments);
+	      } else if (this._currentFrom) {
 	        if (arguments.length > 3) throw new Error('arguments too long');
-	        this.event_on_from.apply(this, arguments);
-	      } else if (this._current_controller) {
+	        this.eventOnFrom.apply(this, arguments);
```

Comment:
__[eslint]__
* Error - Mixed spaces and tabs. (no-mixed-spaces-and-tabs)
* Error - use the spread operator instead of the '.apply()'. (prefer-spread)
* Error - Expected indentation of 2 space characters but found 0. (indent)
* Error - Use the rest parameters instead of 'arguments'. (prefer-rest-params)

<!-- Comment by [SideCI](https://sideci.com) -->